### ISSUE TITLE
Spells don't show up in Spell Specialisation

### ISSUE THREAD

**Issue by Truinto** [Vek17/TabletopTweaks-Core#6]
Spells from mods don't show up in parameterized spell selection, which in the base game are literally just SpellSpecialization First and 1 through 20 (yes, that one feat needs a lot of extra blueprints).

Noticed this effects on spells from pheonix99/TomeOfTheFirebird

I propose you add logic here:
https://github.com/Vek17/TabletopTweaks-Core/blob/c5984b1c62c34db8012d2ce47316d3f0b2173fd8/TabletopTweaks-Core/Utilities/SpellTools.cs#L26

Something similar to mine.
https://github.com/Truinto/DarkCodex/blob/923d466889a2674ff722803dd1255eb24ef57907/CodexLib/Helper.cs#L2315

In short, add the spell's blueprint to BlueprintParameterVariants.

**Comment by Vek17** [comment]
This will be shipped in the actual release for DLC 3, has been implemented in Dev now. Thanks for the issue, had meant to do this ages ago.
